### ISSUE TITLE
Assertion in `addEventListener` does not line up with what is advertised in the docs

### ISSUE THREAD

**Issue by sclaxton** [ember-lifeline/ember-lifeline#614]
https://github.com/ember-lifeline/ember-lifeline/blob/803117c4b6899f181186d694e8749ec414956b8b/addon/dom-event-listeners.ts#L99

This assertion seems to assert that the second argument must be a DOM element and not a string, etc. However, this is not in line with the [current docs](https://ember-lifeline.github.io/ember-lifeline/docs/add-event-listener):

>addEventListener will provide the runloop and remove the listener when destroy is called, provided runDisposables is called. addEventListener provides several ways to specify an element:
>
>```// Attach to an element inside this component
>addEventListener(this, '.someClass', 'scroll', fn);
>
>// Attach to the component's element itself
>addEventListener(this, 'scroll', fn);
>
>// Attach to a DOM node
>addEventListener(this, document.body, 'click', fn);
>
>// Attach to window
>addEventListener(this, window, 'scroll', fn);```

**Comment by rwjblue** [comment]
Yep, definitely out of whack here thanks for reporting! Mind sending a PR in for the README update?
